src/components/CustomTextFieldGroup/CustomTextFieldGroup.tsx

```diff
@@ -0,0 +1,22 @@
+import TextField, {
+  TextFieldVariants,
+} from '@mui/material/TextField';
+import { LoginTextFieldProps } from '../types';
+
+const CustomTextFieldGroup = ({
+  id,
+  label,
+  variant,
+  type,
+}: LoginTextFieldProps) => {
+  return (
+    <TextField
+      id={id}
+      label={label}
+      variant={variant as TextFieldVariants}
```

Comment:
non-blocking: set the type of `variant` prop as TextFieldvariants instead of using the type alias